http-filter-example/http_filter.cc

```diff
@@ -0,0 +1,87 @@
+#include <string>
```

Comment:
Can you split this file to 3 like echo2 filter?

- http_filter.h
- http_filter.cc
- http_filter_config.cc